### ISSUE TITLE
Feature Request: Save .torrent files instead of .magnet to Blackhole directory

### ISSUE THREAD

**Issue by Concept211** [Jackett/Jackett#9251]
Currently, when clicking on the Blackhole buttons it downloads the magnet link as a .magnet file. It would be useful if it could download the .torrent file instead.

**Comment by andekande** [comment]
+1 for example transmission can only import .torrent from a watch folder

**Comment by Concept211** [comment]
> +1 for example transmission can only import .torrent from a watch folder

Exactly. That's the reason I asked for this. I think nearly all torrent clients only monitor .torrent files in their watch folders.

**Comment by DavidJares** [comment]
Please add this feature. It seems there are some online converters, therefore, it should be "easily" manageable to add this feature? Thank you for your work guys!

**Comment by andekande** [comment]
Investigated that topic a bit.
A .torrent container holds much more information than a magnet link contains (for example all the single files and theirs size to download)
So actually a connection to the torrent network is necessary to retrieve the missing metadata.
The Tool that does this is programmatically is https://www.libtorrent.org/ and actually is at the heart of many BitTorrent clients. Like this, the client actually does the lookup of what a magnet link points to.

For the time being, a valid solution for Transmission users is, to run its own command line tool `transmission-remote` that handles magnet links, scheduled by chron or similar:

```
[loop over .magnet files in folder]
[for each file]   
   [read file content] (something like <magnet:?xt=urn:btih:769...>)   
   .\transmission-remote" localhost:9091 --add "[filecontent]"   
   [(re-)move .magnet file]   
```

Lookup google or github gists to find those scripts for your shell (linux and windows alike)

**Comment by MichelBaie** [comment]
Gives error for some others services like https://github.com/thcolin/sensorr/ so please add possibility to have only .torrent files 😐

**Comment by PaarthShah** [comment]
Any change on this?

**Comment by SjoerdV** [comment]
This script worked great
https://github.com/weirdcrap/magnet2torrent
